File: Jumlah dan Produk Barang/Jumlah-dan-Produk-Barang.py
batas = int (input ())

jumlah = 0
produk = 1

for i in range (1, batas):
    if i % 3 == 0 or i % 5 == 0:
        jumlah += i
        produk *= i

print (jumlah)
print (produk)

File: Jumlah dan Produk Barang/Jumlah-dan-Produk-Barang.c
#include <stdio.h>

int main ()
{
    int batas = 0, jumlah = 0, produk = 1;

    scanf ("%d", &batas);

    for (int i = 1; i < batas; i++)
    {
        if (i % 3 == 0 || i % 5 == 0)
        {
            jumlah += i;
            produk *= i;
        }
    }

    printf ("%d\n%d", jumlah, produk);

    return 0;
}